Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -495,6 +502,45 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
 
             return Task.FromResult(item);
         }
+        private void AddParentModules(string module, string path) {
+            if (ModulePath.PythonVersionRequiresInitPyFiles(_analyzer.LanguageVersion) ||
+                string.IsNullOrEmpty(_rootDir)) {
+                return;
+            }
+
+            var dir = Path.GetDirectoryName(path);
```

Comment:
Do we care if path is not valid? I.e. is there a case path with invalid characters or form is useful still?

Reply:
We care because it raises an exception that you aren't handling :) `GetParent` won't raise until we try and use the path for actual I/O, where we'll be handling those exceptions anyway.